packages/flutter/lib/src/foundation/stack_frame.dart

```diff
@@ -105,7 +105,7 @@ class StackFrame {
   static StackFrame _parseWebDebugFrame(String line) {
     final bool hasPackage = line.startsWith('package');
     final RegExp parser = hasPackage
-        ? RegExp(r'^(package:.+) (\d+):(\d+)\s+(.+)$')
+        ? RegExp(r'^(package.+) (\d+):(\d+)\s+(.+)$')
```

Comment:
Consider adding a comment for this regexp. Separately, where would 'package' not be followed by ':' ?

Reply:
I filled a bug on it here with more details: https://github.com/flutter/flutter/issues/52685

Reply:
kk. Maybe link to that issue in a comment here.

Reply:
Done